lib/utils/cli.go

```diff
@@ -211,40 +211,43 @@ func formatErrorWriter(err error, w io.Writer) {
 }
 
 func formatCertError(err error) string {
+	const unknownAuthority = `WARNING:
+The proxy you are connecting to has presented a certificate signed by a
```

Comment:
Is it intentional that some of the new lines are removed in both warnings? Also seems each line had two extra spaces at the beginning.

Reply:
[8f4e8d2](https://github.com/gravitational/teleport/pull/14985/commits/8f4e8d222ac836315e1dfff7e96afa769aeda246)